### PR TITLE
{compiler}[system/system] PGI v20.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
PGI 20 is LLVM/Clang based not GCC. How should we deal with that?
It uses LLVM 9.0 internal.

So it doesn't really need anything but binutils and contains:
OpenBLAS 0.3.3
OpenMPI 3.1.3
ScaLAPACK 2.0.2
Cuda 10.1 and 10.2

**Comment by akesandgren** [comment]
Apparently the release notes lie, the install script says it's Scalapack 2.1.0

**Comment by akesandgren** [comment]
However, the install script for the llvm based version still checks the GCC version and adds the normal config things in localrc.

Also we need to update pgi.py since there is no longer a pgf77 for the sanity check.

**Comment by akesandgren** [comment]
Anyhow, the current pgi.py and EC seem to make something sane. It doesn't install the mpi and scalapack parts. But it does install libblas and liblapack, libblas is openblas (fat) while lapack seem to be netlib based

File: easybuild/easyconfigs/p/PGI/PGI-20.1-GCC-9.2.0-2.32.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+description = "C, C++ and Fortran compilers from The Portland Group - PGI"
+
+toolchain = SYSTEM
+
+sources = ['pgilinux-20%(version_major)s-%(version_major)s%(version_minor)s-x86-64.tar.gz']
+checksums = ['996c76867062a41361fcac16784c3b39fdb744e6a8c18d436ec224f062ada4d8']
+
+local_gccver = '9.2.0'
+local_binutilsver = '2.32'
+versionsuffix = '-GCC-%s-%s' % (local_gccver, local_binutilsver)
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/PGI/PGI-20.1-GCC-9.2.0-2.32.eb]: Do we want to keep it this way, or use GCC-9.2.0 without specifying binutils version like we do for most everything else now??

Comment (boegel) [reply easybuild/easyconfigs/p/PGI/PGI-20.1-GCC-9.2.0-2.32.eb]: Yeah, I should change that part...

**Comment by boegel** [comment]
@akesandgren Is there an easy way to filter out the OpenBLAS/LAPACK that is included with PGI? Since we roll with our own stuff on top of the compiler, it doesn't make much sense to include it...

**Comment by akesandgren** [comment]
Didn't see one when I made a hand-install but i can check again.
They are called libblas and liblapack so they won't interfere with openblas as such

File: easybuild/easyconfigs/p/PGI/PGI-20.1-GCC-9.2.0-2.32.eb (hunk 1 above)
Comment (akesandgren) [reply easybuild/easyconfigs/p/PGI/PGI-20.1-GCC-9.2.0-2.32.eb]: And do we even want to drop even the GCC suffix? we did for iccifort... and only have a suffix for exceptions

Comment (boegel) [reply easybuild/easyconfigs/p/PGI/PGI-20.1-GCC-9.2.0-2.32.eb]: PGI isn't part (yet?) of the commonly used toolchains yet though, while `iccifort` is...

Comment (akesandgren) [reply easybuild/easyconfigs/p/PGI/PGI-20.1-GCC-9.2.0-2.32.eb]: No, but that has been a target of mine for a while. I just never managed to find enough time...

Comment (akesandgren) [reply easybuild/easyconfigs/p/PGI/PGI-20.1-GCC-9.2.0-2.32.eb]: But dropping the binutils suffix is good enough to start with.
We need 20.1 installed next week so deciding on the suffix today or tomorrow would be good.

**Comment by akesandgren** [comment]
Double checked. There is no question or easy way to make it not install the libblas/lapack

**Comment by boegel** [comment]
Not planning to pursue this, so closing...

**Pull request closed without merging**